mesheryctl/pkg/utils/component.go

```diff
@@ -152,9 +153,9 @@ func (mch *ComponentCSVHelper) ParseComponentsSheet() error {
 				mch.Components[data.Registrant][data.Model] = make([]ComponentCSV, 0)
 			}
 			mch.Components[data.Registrant][data.Model] = append(mch.Components[data.Registrant][data.Model], data)
-			fmt.Printf("Reading Component: %s for Modal: %s from Registrant: %s\n", data.Component, data.Model, data.Registrant)
+			Log.Info(fmt.Sprintf("Reading Component: %s for Model: %s from Registrant: %s\n", data.Component, data.Model, data.Registrant))
```

Comment:
```suggestion
			Log.Info(fmt.Sprintf("Reading Registrant [%s] Model [%s] Component [%s]\n", data. Registrant, data.Model, data. Component))
```

Ordering the data this way will be easier to scan with your eyes.